src/us/kbase/test/auth2/lib/storage/mongo/MongoStorageStartUpTest.java

```diff
@@ -1,14 +1,11 @@
 package us.kbase.test.auth2.lib.storage.mongo;
 
 import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
 
+import static org.junit.Assert.*;
```

Comment:
Please don't use wildcard imports, they pollute the namespace with unused names and make it difficult to figure out where a name came from.

Reply:
👍

Reply:
:+1: